launcher/server/server.go

```diff
@@ -0,0 +1,96 @@
+// Package server implements a server to be run in the launcher.
+// Used for communicate between the host/launcher and the container.
+package server
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/google/go-tpm-tools/launcher/agent"
+)
+
+const (
+	// Copy the const from container_runner.go to avoid cyclic
+	// import dependency. This can be removed once the getDefaultToken
+	// can call Attest() directly.
+	hostLauncherPath             = "/tmp/container_launcher/"
+	attestationVerifierTokenFile = "attestation_verifier_claims_token"
+)
+
+type attestHandler struct {
```

Comment:
It seems like should be able to have the fields in attestHandler used by TeeServer. Having an attestHandler complicates a lot of this logic.

Reply:
Another way is to use closure in the New() function, but I think it is more complex and maybe harder to test. So I plan to leave this as here, the logic can be change later when implementing the actual logic.